queries.sql

```diff
@@ -33,3 +33,63 @@ SELECT * from animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
 
 -----------------------------------------------
 -----------------------------------------------
+BEGIN;
+
+UPDATE animals
+SET species = 'unspecified';
+
+ROLLBACK;
```

Comment:
- [ ] Good job so far, :clap: however, according to the project requirement you should `Verify that change was made` before Rollback and after `Rollback` by using select statement. Please resolve that :slightly_smiling_face: